assets/sass/v2/_select-authenticator.scss

```diff
@@ -67,6 +67,13 @@
   display: flex;
   align-items: center;
 
+  .authenticator-usage-text {
+    color: $light-text-color;
+    font-size: 13px;
```

Comment:
do we really need all these styles? 
* `font-size` should be inherited, so it is the same as the authenticator description
* `line-height`: same situation, should be inherited to the same
* `margin-top`: this we might or might not need. I think it looks good without an extra few pixels, so we should remove it. Can you run it by UX please?
* `color`: this we definitely need

Reply:
UX is ok with having just the color.